support/Pipelines/Bbh/FindHorizon.py

```diff
@@ -0,0 +1,231 @@
+            spatial_christoffel_second_kind,
+        ) = interpolate_tensors_to_points(
+            h5_files,
+            subfile_name,
+            observation_id=obs_id,
+            target_points=cartesian_coords(prolonged_strahlkorper),
+            tensor_names=tensor_names,
+            tensor_types=[
+                tnsr.II[DataVector, 3],
+                tnsr.ii[DataVector, 3],
+                tnsr.Ijj[DataVector, 3],
+            ],
+        )
+        status, iter_info = fast_flow.iterate_horizon_finder(
+            strahlkorper,
+            upper_spatial_metric=inv_spatial_metric,
+            extrinsic_curvature=extrinsic_curvature,
+            christoffel_2nd_kind=spatial_christoffel_second_kind,
+        )
+        logger.debug(
+            f"Horizon finder iteration {iter_info.iteration}: {status}."
+            f" {iter_info.r_min:.3f} <= r <= {iter_info.r_max:.3f},"
+            f" {iter_info.min_residual:.2e} <= residual <="
+            f" {iter_info.max_residual:.2e}"
+        )
+        if status == Status.SuccessfulIteration:
+            continue
+        elif int(status) > 0:
+            logger.info(
+                f"Found horizon around {center} with {iter_info.r_min:.3f} <= r"
+                f" <= {iter_info.r_max:.3f}."
+            )
+            break
+        else:
+            raise RuntimeError(f"Horizon finder failed with status {status}.")
+    # Write the horizon to a file and return it
+    if output:
+        assert output_coeffs_subfile or output_coords_subfile, (
+            "Specify either 'output_coeffs_subfile' or 'output_coords_subfile'"
+            " or both."
+        )
+        if output_coeffs_subfile:
+            legend, ylm_data = ylm_legend_and_data(
+                strahlkorper, obs_time, l_max
+            )
+            with spectre_h5.H5File(output, "a") as output_file:
+                datfile = output_file.try_insert_dat(
+                    output_coeffs_subfile, legend, 0
+                )
+                datfile.append(ylm_data)
+        if output_coords_subfile:
+            vol_data = _strahlkorper_vol_data(strahlkorper)
+            with spectre_h5.H5File(output, "a") as output_file:
+                volfile = output_file.try_insert_vol(output_coords_subfile, 0)
+                volfile.write_volume_data(obs_id, obs_time, vol_data)
+    return strahlkorper
+
+
+@click.command(name="find-horizon")
+@open_volfiles_command(
+    obs_id_required=True, multiple_vars=True, vars_required=False
+)
+@click.option(
+    "--l-max",
+    "-l",
+    type=int,
+    required=True,
+    help="Max l-mode for the horizon search.",
+)
+@click.option(
+    "--initial-radius",
+    "-r",
+    type=float,
+    required=True,
+    help="Initial coordinate radius of the horizon.",
+)
+@click.option(
+    "--center",
+    "-C",
+    nargs=3,
+    type=float,
+    required=True,
+    help="Coordinate center of the horizon.",
+)
+@click.option(
+    "--output",
```

Comment:
Should this be required for the CLI? Not sure what the point of omitting it would be here...nothing would happen

Reply:
You get a logging message telling you that a horizon was found and some info about it. I found that useful to check quickly for a horizon in some initial data without having to write a file.